README.md

```
@@ -1,344 +1,309 @@
+  -e AWS_SECRET_ACCESS_KEY=YY \
+  cloudexploit:latest --compliance=pci
 ```
 
-### Ignoring Passing Results
-You can ignore results from output that return an OK status by passing a `--ignore-ok` commandline argument.
+---
 
-### CSV
+# ⚙️ Runtime Examples
+
+### Full Scan
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)